lang/csharp/src/apache/main/Schema/Aliases.cs

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Avro
+{
+    internal static class Aliases
+    {
+        internal static IList<SchemaName> GetSchemaNames(IEnumerable<string> aliases, string enclosingTypeName, string enclosingTypeNamespace)
```

Comment:
Yes, but I would have to add .ToList() in the places that uses it, since they need a list (e.g. in FixedSchema, EnumSchema and RecordSchema). Is that better?

Reply:
You are right. Ignore me :) I was looking at the 

```
public static RecordSchema Create(string name,
            List<Field> fields,
            string space = null,
            IEnumerable<string> aliases = null,
```

and I saw  `IEnumerable<string> aliases` in the arg and totally did not read depeer. Leave it as is.